users/miketronic/config.h

```diff
@@ -0,0 +1,60 @@
+// Copyright 2023 @miketronic -- Mike B <mxb540@gmail.com>
+// SPDX-License-Identifier: GPL-2.0+
+
+#pragma once
```

Comment:
This file looks like it could use a format pass (`qmk format-c users/miketronic/config.h` should take care of that)

Reply:
So to make this update, I would have to do it locally.  Then how do I push the change back to this PR, or do I have to submit a new PR, which is fine but I just to make sure I'm following the correct process.

Reply:
Any changes that you make to the "userspc_add" branch will automatically update the PR.